pipeline/backend/kubernetes/kubernetes.go

```diff
@@ -222,7 +216,7 @@ func (e *kube) WaitStep(ctx context.Context, step *types.Step, taskUUID string)
 	finished := make(chan bool)
 
 	podUpdated := func(old, new any) {
-		pod := new.(*v1.Pod)
+		pod, _ := new.(*v1.Pod)
```

Comment:
well in this case we still need error handling

Reply:
Done